internal/controller/operator/controllers.go

```diff
@@ -29,12 +30,16 @@ import (
 // BindFlags binds package flags to the given flagSet
 func BindFlags(f *flag.FlagSet) {
 	cacheSyncTimeout = f.Duration("controller.cacheSyncTimeout", *cacheSyncTimeout, "controls timeout for caches to be synced.")
-	maxConcurrency = f.Int("controller.maxConcurrentReconciles", *maxConcurrency, "Configures number of concurrent reconciles. It should improve performance for clusters with many objects.")
+	maxConcurrency = f.Int("controller.maxConcurrentReconciles", *maxConcurrency, "configures number of concurrent reconciles. It should improve performance for clusters with many objects.")
+	baseDelay = f.Duration("controller.baseDelay", *baseDelay, "defines a base delay for exponential rate limit")
+	maxDelay = f.Duration("controller.maxDelay", *maxDelay, "defines a max delay for exponential rate limit")
```

Comment:
Could you clarify when we advise users to increase `-controller.baseDelay` and `-controller.maxDelay` and what are the suggested values?
The original `maxDelay: 2m` looks small to me, but I'm not sure we need to expose those values, since it's hard to tell what to set.